podaac/subsetter/subset.py

```diff
@@ -1036,7 +1036,10 @@ def convert_to_datetime(dataset: xr.Dataset, time_vars: list) -> Tuple[xr.Datase
     """
 
     for var in time_vars:
-        start_date = datetime.datetime.strptime("1993-01-01T00:00:00.00", "%Y-%m-%dT%H:%M:%S.%f")
+        if file_extension == 'HDF5':
```

Comment:
Thanks for going through this @danielfromearth. The solution that I'm targeting now is to have a time_converting.py module. Have converting functions for tai93 and GPS time. As of now, we know that xarray doesn't decode times for he5 and HDF5 file extensions. So instead of checking extension having a dictionary of the collections we know need what time conversion? Thoughts?

Reply:
That sounds like it would work. However, thinking about this more makes me worried about hard-coding a dictionary of collection names into the code.  Would it be reasonable to check the first (or first few) time value, and use that to determine whether (and which) conversion is needed?

Reply:
@danielfromearth Yeah, and use the min/max times inputed to get which start date. Will put that into the next PR if that's alright with you.